src/Feed.php

```diff
@@ -26,15 +26,33 @@ public function getFeedResponse()
 
     public function getFeedContent()
     {
-        list($class, $method) = explode('@', $this->feedConfiguration['items']);
+        list($class, $method) = explode('@', $this->getFeedConfigurationItems());
 
-        $items = app($class)->{$method}();
+        $items = app($class)->{$method}($this->getFeedConfigurationItemsFilter());
 
         $meta = ['id' => url($this->feedConfiguration['url']), 'link' => url($this->feedConfiguration['url']), 'title' => $this->feedConfiguration['title'], 'updated' => $this->getLastUpdatedDate($items)];
 
         return view('laravel-feed::feed', compact('meta', 'items'))->render();
     }
 
+    protected function getFeedConfigurationItems()
```

Comment:
Rename this to `getFeedMethod` (or something better to reflect that it's actually the classname and the method).